src/components/EditProfile/edit.css

```diff
@@ -0,0 +1,57 @@
+.edit-form {
```

Comment:
Move styling to less folder. That's where other styles are. You can create a less file for your component and import to index.less